src/components/single-project.tsx

```diff
@@ -96,7 +96,7 @@ export const SingleProject = memo(
             >
                 <BadgeNumber value={`${index + 1}.`} />
 
-                <Link to={nameSlug} tw="inline-block w-full">
+                <Link to={nameSlug || ""} tw="inline-block w-full">
```

Comment:
why this OR operator here?

Reply:
These are issues that did not bother to change all over the project just to lunch the project, some places needs to be revisited